README.md

```diff
@@ -313,6 +313,34 @@ will hopefully enable both quit-if-one-screen and mouse-wheel scrolling.
 
 If scrolling still doesn't work for you, you can try to pass the `-S` option in addition.
 
+### Configuration file
+
+`bat` can use a config file for options to cut down on command line flags, that you want to save for each run. 
+
+#### Location 
+
+The default location is operation system/install location dependent (macOS: `Users/<your_user>/Library/Preferences/bat/config`). To get the path of the current location use `bat --config-file` to print out the location of the current config file path.  
+
+You can change this with the `$BAT_CONFIG_PATH` enviroment variable. 
+
+#### Format
+
+The config file is formatted just as you would pass arguments to the CLI. Here is an example: 
+
+```
+# Make sure that the pager gets executed
+--paging=always
+
+# Always enable coloring (will send ANSI codes to pipe commands)
+--color=always
+
+# Set the theme to "TwoDark"
+--theme="TwoDark"
+
+# Do not display line numbers or borders
+--decorations=never
+```
```

Comment:
As for the contents of this configuration file, I would like to make some changes. I don't think that anyone is going to put `--paging=always`, `--color=always` or `--decorations=never` into their configuration file. I certainly wouldn't advise to do so. Instead, I would suggest something like this:

``` bash
# Set the theme to "TwoDark"
--theme="TwoDark"

# Show line numbers, Git modifications and file header (but no grid)
--style="numbers,changes,header"

# Add mouse scrolling support in less (does not work with older
# versions of "less")
--pager="less -FR"

# Use C++ syntax (instead of C) for .h header files
--map-syntax h:cpp

# Use "gitignore" highlighting for ".ignore" files
--map-syntax .ignore:.gitignore
```